xml/System.IO/File.xml

```diff
@@ -3198,7 +3198,13 @@ The following example moves a file.
         <exception cref="T:System.ArgumentNullException">
           <paramref name="sourceFileName" /> or <paramref name="destFileName" /> is <see langword="null" />.</exception>
         <exception cref="T:System.ArgumentException">.NET Framework and .NET Core versions older than 2.1: <paramref name="sourceFileName" /> or <paramref name="destFileName" /> is a zero-length string, contains only white space, or contains invalid characters. You can query for invalid characters by using the <see cref="M:System.IO.Path.GetInvalidPathChars" /> method.</exception>
-        <exception cref="T:System.UnauthorizedAccessException">The caller does not have the required permission.</exception>
+        <exception cref="T:System.UnauthorizedAccessException">
+        The caller does not have the required permission.
+      
+        -or-
+
+        The Operating System has failed to acquire an exclusive access to the destination file.
```

Comment:
Do you know why this is not an issue on Copy?

Reply:
> Do you know why this is not an issue on Copy?

I don't know for sure, but I suspect that Move is atomic and it requires the exclusive access (the source file gets "deleted" only when the file is successfully moved to the destination) while Copy is not (at the end you have two files)